src/nfa/inclusion.cc

```diff
@@ -71,20 +68,15 @@ bool Mata::Nfa::Algorithms::is_included_antichains(
         if (lhs_bigger.size() > rhs_bigger.size()) { // bigger set cannot be subset
             return false;
         }
-
+        
         //TODO: Can this be done faster using more heuristics? E.g., compare the last elements first ...
         //TODO: Try BDDs! What about some abstractions?
-        return std::includes(rhs_bigger.begin(), rhs_bigger.end(),
-            lhs_bigger.begin(), lhs_bigger.end());
+        return lhs_bigger.IsSubsetOf(rhs_bigger);
```

Comment:
side comment, we should polish the ord vector interface

Reply:
That is planned already. The methods will be renamed properly in later PRs.

Reply:
I also dislike, that the naming of the methods is not `snake_case` but `CamelCase`, looks weird.